plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestBackgroundHiveSplitLoader.java

```diff
@@ -713,7 +713,8 @@ public void testHive2VersionedFullAcidTableFails()
         backgroundHiveSplitLoader.start(hiveSplitSource);
         assertThatThrownBy(() -> drain(hiveSplitSource))
                 .isInstanceOfSatisfying(TrinoException.class, e -> assertEquals(NOT_SUPPORTED.toErrorCode(), e.getErrorCode()))
-                .hasMessage("Hive transactional tables are supported with Hive 3.0 and only after a major compaction has been run");
+                .hasMessage("Hive transactional tables are supported with Hive 3.0. " +
```

Comment:
Since you've touched it maybe you can also change the message from `with Hive 3.0` to `since Hive 3.0` or `with Hive 3.0 and above`. 

Same above.

Reply:
Right, changing to "since"